microsoft-edge/webview2/code-samples-links.md

```diff
@@ -1,213 +1,49 @@
-| WinForms | [Get started with WebView2 in WinForms apps](get-started/winforms.md) | The completed tutorial is at [WebView2Samples > WinForms_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinForms_GettingStarted) |
-| WinUI 2 | [Get started with WebView2 in WinUI 2 (UWP) apps](get-started/winui2.md) | Uses the WinUI 2 NuGet package.  There's isn't a completed tutorial project or directory in the WebView2Samples repo. |
-| WinUI 3 | [Get started with WebView2 in WinUI 3 (Windows App SDK) apps](get-started/winui.md) | The completed tutorial is at [WebView2Samples > WinUI3_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted) |
-| WPF | [Get started with WebView2 in WPF apps](get-started/wpf.md) | The completed tutorial is at [WebView2Samples > WPF_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WPF_GettingStarted) |
-
-
-<!-- ====================================================================== -->
-## The Win32 and multi-project WebView2Samples.sln file
-
-The `WebView2Samples` repo includes samples that demonstrate the WebView2 control and the WebView2 API, for Win32, WPF, WinForms, and WinUI.  These samples are hybrid applications that use the Microsoft Edge WebView2 control.
-
-1. In a separate window or tab, see [WebView2Samples repo (README page)](https://github.com/MicrosoftEdge/WebView2Samples).
-
-1. Download or clone the `WebView2Samples` repo, as described in [Download the WebView2 samples repo](how-to/machine-setup.md#download-the-webview2samples-repo) or [Clone the WebView2 samples repo](how-to/machine-setup.md#clone-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.
-
-1. In your resulting, local copy of the repo directory structure, find `*.sln` files.
+These articles provide the specific steps to set up your machine and build, run, and update each sample.
 
-1. Open one of the `.sln` files.  For example, open your local copy of the multi-platform solution file [WebView2Samples/SampleApps/WebView2Samples.sln](https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WebView2Samples.sln) (downloaded as path `WebView2Samples-main/SampleApps/WebView2Samples.sln`) in Microsoft Visual Studio.  When you open that solution file in Visual Studio, **Solution Explorer** contains the following WebView2 samples, as projects:
 
-   ![Solution Explorer for the WebView2Samples repo, showing the WebView2 samples as projects.](media/solution-file-webview2samples.png)
+#### Sample apps vs. completed Getting Started tutorials
 
-This particular `.sln` file doesn't include the Get Started projects, which are separate `.sln` files, one per platform.
+For some frameworks or platforms, the samples repo contains both a completed Getting Started tutorial and a sample app.  The sample apps demonstrate more features than the completed Getting Started tutorials.
```

Comment:
The sample apps are home to sample code in the reference docs and conceptual docs. These apps showcase more of our APIs than then Getting Started tutorials.